gtsam/geometry/Cal3DS2_Base.h

```diff
@@ -39,22 +41,43 @@ namespace gtsam {
 class GTSAM_EXPORT Cal3DS2_Base {
 
 protected:
-
-  double fx_, fy_, s_, u0_, v0_ ; // focal length, skew and principal point
-  double k1_, k2_ ; // radial 2nd-order and 4th-order
-  double p1_, p2_ ; // tangential distortion
+ double fx_, fy_, s_, u0_, v0_;  // focal length, skew and principal point
```

Comment:
`///<` is doxygen one-liner convention

Reply:
I'm done refactoring all the calibration models. This has been taken care of in that PR.